reporter/src/funTest/kotlin/reporters/ExcelReporterFunTest.kt

```diff
@@ -40,13 +39,43 @@ class ExcelReporterFunTest : WordSpec({
             val ortResult = readOrtResult(
                 "../scanner/src/funTest/assets/file-counter-expected-output-for-analyzer-result.yml"
             )
-            val sheet = ExcelReporter().generateReport(ReporterInput(ortResult), outputDir).single()
-            val actualWorkbook = WorkbookFactory.create(sheet)
 
-            val expectedFile = File("src/funTest/assets/file-counter-expected-scan-report.xlsx")
-            val expectedWorkbook = WorkbookFactory.create(expectedFile)
+            val report = ExcelReporter().generateReport(ReporterInput(ortResult), outputDir).single()
+            val actualWorkbook = WorkbookFactory.create(report)
+            val actualSheetNames = actualWorkbook.sheetIterator().asSequence().map { it.sheetName }.toList()
+
+            val asset = File("src/funTest/assets/file-counter-expected-scan-report.xlsx")
+            val expectedWorkbook = WorkbookFactory.create(asset)
+            val expectedSheetNames = expectedWorkbook.sheetIterator().asSequence().map { it.sheetName }.toList()
+
+            actualSheetNames shouldContainExactly expectedSheetNames
+            actualSheetNames.forAll { sheetName ->
+                val actualSheet = actualWorkbook.getSheet(sheetName)
+                val actualRowNums = actualSheet.rowIterator().asSequence().map { it.rowNum }.toList()
+
+                val expectedSheet = expectedWorkbook.getSheet(sheetName)
+                val expectedRowNums = expectedSheet.rowIterator().asSequence().map { it.rowNum }.toList()
+
+                actualRowNums shouldContainExactly expectedRowNums
+                actualRowNums.forAll { rowNum ->
+                    val actualRow = actualSheet.getRow(rowNum)
+                    val actualColumnIndices = actualRow.cellIterator().asSequence().map { it.columnIndex }.toList()
+
+                    val expectedRow = expectedSheet.getRow(rowNum)
+                    val expectedColumnIndices = expectedRow.cellIterator().asSequence().map { it.columnIndex }.toList()
+
+                    actualColumnIndices shouldContainExactly expectedColumnIndices
+                    actualColumnIndices.forAll { columnIndex ->
```

Comment:
What do you think about `actualColumnIndices.map { it to rowNum }.forAll { columnIndex, rowNum ->` to make the output in case of a failure contain the `rowNum` in additon to `columnIndex`?

Reply:
Done.